### DISCUSSION TITLE
Option to hide global toc (for landing page)

### DISCUSSION THREAD

**Discussion by chrisjsewell** [pradyunsg/furo#214] [Ideas] [answered]
Similar to https://pradyunsg.me/furo/customisation/toc/, I wonder if it is possible to specify a page(s) where the global toc is hidden by default (not just on small screens)

The use case here would be for landing pages like: https://sphinx-design.readthedocs.io/en/furo-theme/, where I want it to be more “minimalist”, with initial links in the content

**Comment by chrisjsewell** [comment]
the idea is to have a nice landing page like https://code.visualstudio.com/, with a link into the main docs content https://code.visualstudio.com/docs, and using sphinx-design to create the landing page in Markdown/RST, rather than have to do it directly in HTML

**Comment by pradyunsg** [comment, marked as answer]
Thanks for the suggestion!

I feel that if you're hiding the primary sidebar, you might be better off [using `html_additional_pages` at that point](https://pradyunsg.me/furo/customisation/landing-page/). I don't want to provide this knob; since I don't see much value in providing this over allowing authors to write their HTML that builds upon what Furo provides + can include the Furo CSS as well.

This might make sense for themes that want to cover more usecases -- but I don't want Furo to be such a theme. :)
